manifests/Typora/Typora/0.9.87.yaml

```diff
@@ -0,0 +1,15 @@
+Id: Typora.Typora
+Publisher: Typora
+Name: Typora
+Version: 0.9.87
+License: Typora is developed by @LeeAbner. All rights reserved.
+LicenseUrl: http://support.typora.io/Acknowledgement/
+InstallerType: exe
+Installers:
+  - Arch: x64 # enumeration of supported architectures
+    Url: https://typora.io/windows/typora-setup-x64.exe
+    Sha256: ff5b0288c1c563bae4b1d4250d870e2bb758639b727815da45e658a5240a2cc4
+    Switches:
+      Silent: "/VERYSILENT /SUPPRESSMSGBOXES /NORESTART"
+      SilentWithProgress: "/VERYSILENT /SUPPRESSMSGBOXES /NORESTART"
+# ManifestVersion: 0.1.0
```

Comment:
```suggestion
InstallerType: inno
Installers:
  - Arch: x64 # enumeration of supported architectures
    Url: https://typora.io/windows/typora-setup-x64.exe
    Sha256: ff5b0288c1c563bae4b1d4250d870e2bb758639b727815da45e658a5240a2cc4
# ManifestVersion: 0.1.0
```

Typora uses innosetup, so you can use the inno installertype and not need to include the switches.

Reply:
thanks, I get it.